packages/url_launcher/url_launcher/example/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  path: ^1.8.2
```

Comment:
I think `stable` is on 1.8.0 or 1.8.1, which is probably why `pub get` is failing (which on CI is a timeout) in stable analysis.

Reply:
Indeed:

```
Because every version of integration_test from sdk depends on path 1.8.1 and url_launcher_example depends on path ^1.8.2, integration_test from sdk is forbidden.
So, because url_launcher_example depends on integration_test from sdk, version solving failed.
Running "flutter pub get" in example...                                 
pub get failed (1; So, because url_launcher_example depends on integration_test from sdk, version solving failed.)
```